modules/migrate/application/clicommands/PreferencesCommand.php

```diff
@@ -102,4 +104,27 @@ public function indexAction()
 
         Logger::info('Successfully migrated all local user preferences to database');
     }
+
+    private function loadIniFile($filePath, User $user): array
```

Comment:
Not required, but just out of interest: You've added a return type declaration, so you're aware of the PHP 7.x possibilities, why aren't you then using the `string` type declaration for `$filePath` also?

Reply:
You're also not strictly requiring the `User` object, just pass the name.